src/test/java/com/thebois/models/world/WorldTests.java

```diff
@@ -43,6 +46,35 @@ public void worldFind() {
         assertThat(worldObject).isEqualTo(null);
     }
 
+    @Test
+    public void getStructuresTest() {
+        // Arrange
+        final World world = new World(25);
+
+        // Act
+        world.createStructure(new Position(123, 123));
```

Comment:
This line is only for setting up the test. The line below, where `world.getStructures()` is called, is the acting.